app/controllers/photos.js

```diff
@@ -0,0 +1,7 @@
+const { getPhotos, getPhotosByAlbum } = require('../services/albums');
+
+exports.getPhotos = (req, res, next) => {
+  const { albumId } = req.query;
```

Comment:
I think the album id should be a url parameter rather than query parameter and you can make a generic endpoint for all photos separately.
so you go `/albums/:albumId/photos`